README.md

```diff
@@ -62,7 +62,8 @@ sudo apt-get install -y libffi-dev libssl-dev make build-essential libssl-dev zl
 3. Install Python 3.7.3: `pyenv install 3.7.3 && pyenv local 3.7.3`
 4. Check correctly installed: `python -V`
 5. Setup virtualenv: `python3 -m venv venv && source venv/bin/activate`
-6. Install dependencies: `pip install -r requirements.txt`
+6. Install python `wheel` package: `python3 -m pip install wheel`
```

Comment:
What's your best guess of why this is not needed in the Dockerfile? Does one of the packages installed with `apt-get install` include this?

Reply:
My guess is the balena python images include it already, but not 100% sure

Reply:
I was doing local dev setup on ubuntu and that does not use Dockerfile when setting up the project. My guess is that we would probably want to add this in Dockerfile too. I'll check that once I deploy this via docker and update Dockerfile if needed. Since these packages are being fetched from pypi I have a hunch that we would want to update Dockerfile too.

Reply:
The Dockerfile is in charge of the build that is referenced by helium-miner-software and ultimately runs in production miners. We haven't made changes to the Dockerfile recently, so I'd be surprised if we did also need these changes there.

Reply:
@marvinmarnold @kashifpk I think this is why https://www.raspberrypi.com/news/piwheels/

I.e. the pi specific balena builds already have support baked in by default with pip